### PR TITLE
Updates Neith's Holocoat in accordance to the Eridani Rework

### PR DESCRIPTION
**Pull request by VisVirific**
Title

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
Not sure what exactly is going on here? Why does this item have to be removed?

**Comment by VisVirific** [comment]
> Not sure what exactly is going on here? Why does this item have to be removed?

Eridani Rework reworked Oran into.... just an Eridani planet without any name or real characteristics. I'm just following lore and remaining consistent as this holocoat was made in Oran, and since it doesn't exist anymore, then I can't really have it. Nor does Neith Mekesatis even exist

**Comment by NonQueueingMatt** [comment]
> > Not sure what exactly is going on here? Why does this item have to be removed?
> 
> Eridani Rework reworked Oran into.... just an Eridani planet without any name or real characteristics. I'm just following lore and remaining consistent as this holocoat was made in Oran, and since it doesn't exist anymore, then I can't really have it. Nor does Neith Mekesatis even exist

It seems like a bit of an overreaction? Surely your character can just come from one of the planets instead of Oran. Changing your character's origin planet's name in your backstory (by not that much, factually -- the planets are not that different now) is something that is really quite easy to retroactively do.

**Comment by VisVirific** [comment]
!review

**Comment by VisVirific** [comment]
> It seems like a bit of an overreaction? Surely your character can just come from one of the planets instead of Oran. Changing your character's origin planet's name in your backstory (by not that much, factually -- the planets are not that different now) is something that is really quite easy to retroactively do.

Considering that the decision for enhancing an entire human faction's lore with multiple planets to itself has been to dumb it down and effectively eliminate it's planets, something you seem to consider easy to do or accept as "the planets are not that different now", only proves that this isn't an overreaction.

After all, if, say, Dominia or Elyra were all reduced to one page only with their own planets barely receiving one or two paragraphs each, I'd bet there'd be riots. But doing so to Eridani is acceptable. It's now unique in how bland it is as a faction without anything to itself besides the bare trappings of an idea. It's now unique in how little to itself it actually has. It's now unique to be watered down even more be it in lore or in-game code.

I was going to go on in explaining my criticisms to this whole "Rework", but then I figured it wouldn't matter. If Lain, Schwann, and Omicega all considered this to be The Best Decision, then who am I to change that? It's not the first time something has been reworked in this fashion and going into several pages length of criticisms only nets you the badge of Funny Man, with Free Copypasta Meme, and getting trashtalked about Overreacting.

Hence I'm just following the spirit and lore consistency to the Eridani Rework. Dumbing down my character until the name of his home planet barely matters, or even himself for that matter.

**Review by SleepyGemmy — APPROVED** [review]

**Review by Alberyk — APPROVED** [review]
